android/src/oldarch/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -47,6 +47,10 @@ public RNCWebView createViewInstance(ThemedReactContext context) {
         return mRNCWebViewManagerImpl.createViewInstance(context);
     }
 
+    public RNCWebView createViewInstance(ThemedReactContext context, RNCWebView webView) {
+      return mRNCWebViewManagerImpl.createViewInstance(context, webView);
+    }
+
```

Comment:
Is this necessary? wouldn't just people overriding things replace the first one with a custom one?

Reply:
Sorry I didn't see this sooner.

The reason for this was that `mRNCWebViewManagerImpl` is declared `private final`, so subclasses don't have access to call `createViewInstance` on it. This way allows overriding the single-parameter version and then using `super` to invoke the two-parameter version.

Making it `protected` instead of `private` would also have been an option. I'm not sure if there's much advantage to one way or the other. I think I just initially assumed that the use of `private` was an intentional choice and so this would be a less-controversial approach.